velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -260,6 +260,34 @@ struct FromUnixtimeFunction {
   bool isConstantTimeFormat_{false};
 };
 
+template <typename T>
+struct ToUTCTimestampFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
+      out_type<Timestamp>& result,
+      const arg_type<Timestamp>& timestamp,
+      const arg_type<Varchar>& timezone) {
```

Comment:
I think the timezone can be constant in many use cases. It would be nice to add `initialize` method to pre-handle constant timezone for optimization. Same for the other function.

See reference:
https://github.com/facebookincubator/velox/blob/de55ae9453d4b7323f028d2029cff105f03b9f3d/velox/functions/sparksql/DateTimeFunctions.h#L170

Reply:
Done, thanks!